pkg/exporters/slack_test.go

```diff
@@ -126,6 +126,29 @@ func TestNewSlackMessage(t *testing.T) {
 	assert.Equal(t, message.Title, attach.Title)
 }
 
+func TestNewSlackMessageNoIDs(t *testing.T) {
+	message := msg.Message{
+		TitleLink: "https://myvcslink/",
+		Title:     "The title of the message",
+		Body:      "this is the message body",
+		Event: fluxevent.Event{
+			ServiceIDs: []flux.ResourceID{},
```

Comment:
After rebasing, I think you need to update this reference
```suggestion
			ServiceIDs: []resource.ID{},
```